### PR TITLE
Event Getter provide time

### PR DESCRIPTION
**Pull request by dneise**
Based on the last results accomplished by Mario and Max, I understand, we learned that the offsets depend *at least* on:

 * cell_id,
 * (relative) sample_id,
 * delta_t: time since a cell_id was last read out

We might not yet have the dependency of these parameters fully understood, but I strongly believe we should be prepared to let users of the `EventGetter` have easy access to these parameters.

### CODE REVIEW

**Comment by dneise** [comment]
I think this kind of reader could be used like this for example, once the offsets have been found out of course. 

```python
import dragonboard as db

from some_offset_storage import offset

with db.EventGenerator("some/path/file.dat") as eg:
    for event in eg:
        for gain_type, channel in db.io.stop_cell_map:

            g = gain_type
            c = channel

            sample_ids = np.arange(40)
            sc = event.header.stop_cells[g][c]

            calibrated_data = event.data[g][c] - offset(
                channel=channel,
                gain_type=gain_type,
                cell_ids=(sample_ids + sc) % db.io.max_roi,
                sample_ids=sample_ids,
                delta_ts=event.since[g][c],
                temperature=event.temperature, # who knows...
                )
```

File: dragonboard/io.py [hunk 1]
```diff
@@ -125,6 +144,27 @@ def previous(self):
...
+            dtype=[
+                ("low", 'f4', self.roi),
+                ("high", 'f4', self.roi),
+            ]
+        )
+
+        for g, p in stop_cell_map:
+            sc = event_header.stop_cells[g][p]
+
+            since[g][p] = np.roll(self.last_seen[g][p], -sc)[:self.roi]
```
Comment (MaxNoe) [inline dragonboard/io.py]: We learned in #30 that `np.role` is quite expensive. Is it really necessery here?

**Comment by MaxNoe** [comment]
I do not like the `since` name. 

I try to come up with a better Idea:
* `last_readout`
* `since_last_readout`
* `time_since_last_readout`
* `delta_t`
* `delta_t_readout`

I would go for `delta_t`, as we used this quite a lot already.

**Comment by dneise** [comment]
I don't like `delta_t`, because it does only convey, we are looking a time difference, but it does not show, what it refers to.

The fact, that we used it quite a lot in the Telegram chat, does imho not imply it is a good name, but only that it is a short name. Look at the confusion, which was caused by the usage of `offset` in the chat yesterday. 

I don't like `last_readout` because this sounds like it is not the difference between the last readout and now, but the time of the last readout itself.

----
Let me explain, how I arrived at `since`:

`time_since_last_readout` is ultimatively correct, albeit long. I guess we do agree on that.

In order to create a *shorter* but still descriptive name, I think we can drop `time`, because the word `since` already means [between a past time or event and the present](http://www.dictionary.com/browse/since) when used as a preposition.

So we arrived at `since_last_readout`. 

Now I think we can drop even the `last_readout` part, because the last readout of a cell is the only *interesting event in the past* of a cell. If this assumption is not correct, we can not drop the `last_readout`. However I would like to know what other event in the past of a cell is interesting for us?

**Pull request merged**